Lib/test/test_mmap.py

```diff
@@ -887,6 +886,92 @@ def test_resize_succeeds_with_error_for_second_named_mapping(self):
         self.assertEqual(m1[:data_length], data)
         self.assertEqual(m2[:data_length], data)
 
+    def test_mmap_closed_by_int_scenarios(self):
+        """
+        gh-103987: Test that mmap objects raise ValueError
+                for accidentally closed mmap files
```

Comment:
```suggestion
                for closed mmap files
```
Pretty sure we're doing it on purpose